CHANGELOG.md

```diff
@@ -73,6 +73,9 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Check for the existence of the runtime executable prefix, to avoid
   issues when running under Slurm's srun. If it doesn't exist, fall
   back to the compile-time prefix.
+- If a `remote` is defined both globally (i.e. system-wide) and
+  individually, change `apptainer remote` commands to print an info message
+  instead of exiting with a fatal error.
```

Comment:
Lets include that it uses the individuals configuration over the global one in this as well